zerver/openapi/zulip.yaml

```diff
@@ -6714,12 +6714,18 @@ paths:
                           Present if `realm` is present in `fetch_event_types`.
 
                           The policy for which users can create bot users in this organization.
-                      realm_create_stream_policy:
+                      realm_create_public_stream_policy:
                         type: integer
                         description: |
                           Present if `realm` is present in `fetch_event_types`.
 
-                          The policy for which users can create streams in this organization.
+                          The policy for which users can create public streams in this organization.
+                      realm_create_private_stream_policy:
+                        type: integer
+                        description: |
+                          Present if `realm` is present in `fetch_event_types`.
+
+                          The policy for which users can create private streams in this organization.
```

Comment:
These need to document the changes using a `**Changes**` entry -- search this file for plenty of examples.

I'd write: "Changes: Before Zulip 4.0 (feature level x) split it, the realm_create_stream_policy field controlled creation of both public and private streams".